tools/docker-run.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
```

Comment:
nit: in your commit message, you could provide a bit more details on this new script